Sources/BraveUI/UIKit/UIKitPopupView.swift

```diff
@@ -16,30 +16,30 @@ enum PopupViewDismissType: Int {
   case scaleDown
 }
 
-enum PopupViewShowType: Int {
+public enum PopupViewShowType: Int {
   case normal
   case flyUp
   case flyDown
 }
 
-enum PopupViewAlignment: Int {
+public enum PopupViewAlignment: Int {
   case top
   case middle
   case bottom
 }
 
-enum PopupViewStyle: Int {
+public enum PopupViewStyle: Int {
   case dialog
   case sheet
 }
 
-protocol PopupViewDelegate {
-  func popupViewDidShow(_ popupView: PopupView)
-  func popupViewShouldDismiss(_ popupView: PopupView) -> Bool
-  func popupViewDidDismiss(_ popupView: PopupView)
+public protocol PopupViewDelegate {
+  func popupViewDidShow(_ popupView: UIKitPopupView)
+  func popupViewShouldDismiss(_ popupView: UIKitPopupView) -> Bool
+  func popupViewDidDismiss(_ popupView: UIKitPopupView)
 }
 
-class PopupView: UIView, UIGestureRecognizerDelegate {
+open class UIKitPopupView: UIView, UIGestureRecognizerDelegate {
```

Comment:
There's `PopupView` written in SwiftUI already. I renamed it to something basic, did not want to go too hard on this refactor. Also created `UIKit` folder in `BraveUI` and put some views I used for my BraveVPN target there, we can make it better later imo